docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md

```diff
@@ -0,0 +1,153 @@
+---
+sidebar_label: "Define Profile Variables"
+title: "Define Profile Variables"
+description: "Learn what cluster profile variables are and how to use them."
+sidebar_position: 40
+tags: ["profiles", "cluster profiles"]
+---
+
+Cluster profile variables enable you to create placeholders for parameters in profile layer configurations, which you
+can then populate for individual clusters during deployment. Meaning you can use a single cluster profile to deploy
+multiple clusters with unique requirements for security, networking, resource allocation, and so on.
+
+:::preview
+
+This is a Tech Preview feature currently available in the
+[Edge Management Console](/clusters/edge/edge-management-console/). Do not use this feature in production workloads, as
+it is subject to change.
+
+:::
+
+When defining a cluster profile variable, you can set specific constraints on the expected values, such as format,
+optionality, masking, and so on, to ensure scalable, error-free cluster deployments.
+
+:::info
+
+You can use profile variables with any number of packs, manifests, and Helm charts, but only in the scope of their
+parent cluster profile. If you want to create placeholders to use across different cluster profiles, consider using
+[Palette Macros](/clusters/cluster-management/macros/).
+
+:::
+
+This guide explains how you can define and manage cluster profile variables.
+
+## Prerequisites
+
+- The `clusterProfile.create` and `clusterProfile.update` permissions to create and update cluster profiles. Refer to
+  [Roles and Permissions](/user-management/palette-rbac/project-scope-roles-permissions#cluster-profile-admin) for more
+  information.
+
+- An in-progress or already created cluster profile. The cluster profile must either have the **Edge Native**
+  infrastructure or be an edge-based add-on profile.
+
+## Define Profile Variables
+
+:::warning
```

Comment:
We seem to have quite a few limitations, maybe should add a limitation section. That way, we can keep all of these together, such as this warning, which is only available to Edge, etc.

Reply:
Yup, makes perfect sense, added a separate section with limitations 🌷